crossdc-producer/src/main/java/org/apache/solr/update/processor/MirroringUpdateRequestProcessorFactory.java

```diff
@@ -249,8 +249,10 @@ public UpdateRequestProcessor getInstance(final SolrQueryRequest req, final Solr
             log.trace("Create MirroringUpdateProcessor with mirroredParams={}", mirroredParams);
         }
 
+        String defaultDefaultDBQMethod = conf.get(KafkaCrossDcConf.DEFAULT_DBQ_METHOD);
```

Comment:
[0] Dumb question, but should all these variable names really start with "defaultDefault" instead of just "default"?

Reply:
Yeah, in this case at least it should just be one default.